Source/WebKit/UIProcess/API/Cocoa/_WKWebAuthenticationPanel.mm

```diff
@@ -84,7 +84,7 @@ static inline void updateQueryForGroupIfNecessary(NSMutableDictionary *dictionar
 }
 #endif
 
-static RetainPtr<NSData> produceClientDataJson(_WKWebAuthenticationType type, NSData *challenge, NSString *origin)
+static RetainPtr<NSData> produceClientDataJson(_WKWebAuthenticationType type, NSData *challenge, NSString *origin, std::optional<WTF::String> topOrigin, BOOL crossOrigin)
```

Comment:
No optional.

Reply:
I recommend changing `BOOL crossOrigin` to `bool isCrossOrigin`. We use a prefix for boolean variables in WebKit and we don't strictly have to use ObjC types here.